lib/beaker-pe/install/pe_utils.rb

```diff
@@ -534,6 +551,7 @@ def do_install hosts, opts = {}
           # detect the kind of install we're doing
           install_type = determine_install_type(hosts, opts)
           verify_network_resources(hosts, options[:net_diag_hosts])
+          verify_vm_resources(hosts)
```

Comment:
We should limit this to just pe infrastructure, since that is what is failing to install PE. (also not sure how/if top works on powershell)

Reply:
Right, I'll restrict it

Reply:
No `top` as such in PowerShell, but you can do something like https://superuser.com/a/512206

Reply:
Thanks for the idea @Dakta , but I'm not going to worry about Windows for now, since we don't (yet?) support Windows as a PE Infrastructure platform.